addons/inkgd/editor/ink_dock.tscn

```diff
@@ -86,123 +105,119 @@ size_flags_horizontal = 3
 margin_right = 1881.0
 margin_bottom = 24.0
 size_flags_horizontal = 3
-text = "/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono"
 
 [node name="MonoDialogButton" type="Button" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer/MonoHBoxContainer"]
 margin_left = 1885.0
 margin_right = 1913.0
 margin_bottom = 24.0
 
 [node name="ExecutableLabel" type="Label" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer"]
-margin_top = 28.0
-margin_right = 1913.0
-margin_bottom = 52.0
+margin_right = 953.0
+margin_bottom = 24.0
 size_flags_horizontal = 3
 size_flags_vertical = 1
 text = "Executable"
 
 [node name="ExecutableHBoxContainer" type="HBoxContainer" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer"]
-editor/display_folded = true
-margin_left = 1917.0
-margin_top = 28.0
-margin_right = 3830.0
-margin_bottom = 52.0
+margin_left = 957.0
+margin_right = 1910.0
+margin_bottom = 24.0
 size_flags_horizontal = 3
 
 [node name="ExecutableLineEdit" type="LineEdit" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer/ExecutableHBoxContainer"]
-margin_right = 1881.0
+margin_right = 921.0
 margin_bottom = 24.0
 size_flags_horizontal = 3
-text = "/usr/local/Cellar/inklecate/0.8.2/libexec/inklecate.exe"
 
 [node name="ExecutableDialogButton" type="Button" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer/ExecutableHBoxContainer"]
-margin_left = 1885.0
-margin_right = 1913.0
+margin_left = 925.0
+margin_right = 953.0
 margin_bottom = 24.0
+icon = SubResource( 2 )
 
 [node name="SourceFileLabel" type="Label" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer"]
-margin_top = 56.0
-margin_right = 1913.0
-margin_bottom = 80.0
+margin_top = 28.0
+margin_right = 953.0
+margin_bottom = 52.0
 size_flags_horizontal = 3
 size_flags_vertical = 1
 text = "Source File"
 
 [node name="SourceFileHBoxContainer" type="HBoxContainer" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer"]
-editor/display_folded = true
-margin_left = 1917.0
-margin_top = 56.0
-margin_right = 3830.0
-margin_bottom = 80.0
+margin_left = 957.0
+margin_top = 28.0
+margin_right = 1910.0
+margin_bottom = 52.0
 size_flags_horizontal = 3
 
 [node name="SourceFileLineEdit" type="LineEdit" parent="VSplitContainer/MainVBoxContainer/Panel/MarginContainer/ScrollContainer/VBoxContainer/GridContainer/SourceFileHBoxContainer"]
-margin_right = 1881.0
+margin_right = 921.0
 margin_bottom = 24.0
 size_flags_horizontal = 3
-text = "/Users/fred/Documents/Development/Godot/MollysFork/assets/stories/ink/main.ink"
```

Comment:
Ah, great that you removed my mess!

Reply:
Huh... That's interesting. I don't think I changed it as a scene, but I did change it in the docked version of the scene, that's built into the editor. I think if I have the scene open as a tab and another version docked, any changes that happen to docked version, like writing into the lineedit field or choosing a folder, then the scene changes as well. I'll have to keep an eye on this.

Reply:
> I'll have to keep an eye on this.

So will I, because that's how I introduced a hardcoded path by mistake.

> I don't think I changed it as a scene, but I did change it in the docked version of the scene, that's built into the editor.

No worries, that's good to know. We can figure this later, it's resolved for now!